www/js/app.js

```diff
@@ -1143,6 +1166,11 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
                 if (docBody) {
                     docBody.addEventListener('dragover', handleIframeDragover);
                     docBody.addEventListener('drop', handleIframeDrop);
+                    docBody.addEventListener('keydown',function(e) {
```

Comment:
It can be confusing to put this code here : as all the lines above handle the drag-and-drop feature, one could tend to think this piece of code is also to handle drag-and-drop.
I suggest to put that outside this code block, and add a comment above to explain the purpose. Never mind if you duplicate the `if (docBody)`

Reply:
Done.